hypervisor/arch/x86/configs/nuc7i7dnb/platform_acpi_info.h

```diff
@@ -39,6 +39,9 @@
 #define RESET_REGISTER_SPACE_ID SPACE_SYSTEM_IO
 #define RESET_REGISTER_VALUE    0x6U
 
+/* PCI mmcfg base of MCFG */
+#define DEFAULT_PCI_MMCFG_BASE	0xE0000000UL
```

Comment:
Is the default value also platform specific? Otherwise, we don't need to put here

Reply:
yes, the value is for NUC7i7DNB only. Current code only provided a valid platform_acpi_info.h for NUC7 under configs/nuci7dnb/ folder. When CONFIG_ACPI_PARSE_ENABLED is not set, hypervisor will reference this value directly. For other boards, we should use acrn-config tool to generate DEFAULT_PCI_MMCFG_BASE macro in board specific platform_acpi_info.h;

When CONFIG_ACPI_PARSE_ENALBED is set, the pci_mmcfg_base will be overridden by acpi_fixup();